eo-maven-plugin/src/test/java/org/eolang/maven/it/SnippetTestCase.java

```diff
@@ -301,18 +308,111 @@ private static String jdkExecutable(final String name) {
      * @return Classpath.
      */
     private static String classpath() {
-        return String.format(
-            ".%s%s",
-            File.pathSeparatorChar,
-            System.getProperty(
-                "runtime.jar",
-                Paths.get(System.getProperty("user.home")).resolve(
-                    String.format(
-                        ".m2/repository/org/eolang/eo-runtime/%s/eo-runtime-%1$s.jar",
-                        "1.0-SNAPSHOT"
-                    )
-                ).toString()
-            )
-        );
+        final String home = System.getProperty("user.home");
+        try {
+            if (SnippetTestCase.isNotRealPath(home)) {
```

Comment:
@volodya-lombrozo check this out: https://www.yegor256.com/2015/01/21/if-then-throw-else.html